packages/account/src/Sections/Assessment/TradingAssessment/trading-assessment.jsx

```diff
@@ -0,0 +1,262 @@
+            });
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    const handleSubmit = async values => {
+        setFormData(values);
+        setIsBtnLoading(false);
+        setIsSubmitSuccess(true);
+        if (values.risk_tolerance === 'No') {
+            setShouldAcceptRisk(true);
+        } else {
+            setIsBtnLoading(true);
+            const form_payload = {
+                trading_experience_regulated: {
+                    cfd_experience: values.cfd_experience,
+                    cfd_frequency: values.cfd_frequency,
+                    cfd_trading_definition: values.cfd_trading_definition,
+                    leverage_impact_trading: values.leverage_impact_trading,
+                    leverage_trading_high_risk_stop_loss: values.leverage_trading_high_risk_stop_loss,
+                    required_initial_margin: values.required_initial_margin,
+                    risk_tolerance: values.risk_tolerance,
+                    source_of_experience: values.source_of_experience,
+                    trading_experience_financial_instruments: values.trading_experience_financial_instruments,
+                    trading_frequency_financial_instruments: values.trading_frequency_financial_instruments,
+                },
+            };
+            const data = await setFinancialAndTradingAssessment(form_payload);
+            const { trading_score } = data?.set_financial_assessment;
+            if (trading_score === 0 && !should_accept_risk) {
+                setShouldShowWarningModal(true);
+            }
+            WS.authorized.storage.getFinancialAssessment().then(res_data => {
+                setInitialFormValues(res_data.get_financial_assessment);
+            });
+            setIsBtnLoading(false);
+            setTimeout(() => setIsSubmitSuccess(false), 10000);
+        }
+    };
+
+    const handleAcceptRisk = () => {
+        handleSubmit({ ...form_data, risk_tolerance: 'Yes' });
+        setShouldAcceptRisk(false);
+        setIsBtnLoading(false);
+        setIsSubmitSuccess(false);
+    };
+
+    if (is_loading) return <Loading is_fullscreen={false} className='account__initial-loader' />;
+    if (should_accept_risk) {
+        return (
+            <RiskToleranceWarningModal
+                show_risk_modal
+                title={localize('Risk Tolerance Warning')}
+                button_text={localize('Yes, I understand the risk.')}
+                onClick={handleAcceptRisk}
+                body_content={
+                    <Localize
+                        i18n_default_text='CFDs and other financial instruments come with a high risk of losing money rapidly due to leverage. You should consider whether you understand how CFDs and other financial instruments work and whether you can afford to take the high risk of losing your money. <0/><0/> To continue, you must confirm that you understand your capital is at risk.'
```

Comment:
it seems here is a typo: do we need here two closing tags? </0>

Reply:
I think they're adding two breaks, so it should have two self-closing tags which is fine

Reply:
Yea it is intended to have two because we wanted a bigger gap in between the text.